doc/mgr/dashboard.rst

```diff
@@ -500,6 +500,13 @@ on your preferred hosts, proceed with the following steps.
       [security]
       allow_embedding = true
 
+Enabling RBD-Image monitoring
+"""""""""""""""""""""""""""""
+
+Due to performance reasons, monitoring of RBD images is disabled by default. For
+more information please see :ref:`prometheus-rbd-io-statistics`. If disabled,
```

Comment:
Given that from dashboard back-end we can know if that setting is enabled or not, would it make sense to hide these dashboards in case they're disabled? Or at least display some kind of contextual info indicating this requirement?

Reply:
@epuertat I discussed this with @p-se and we agreed this would be a good enhancement that we wanted to submit in a follow-up PR.

Reply:
ok thanks!